_includes/checkout-v3-resource-models.md

```diff
@@ -870,7 +872,9 @@ Content-Type: application/json
 | └─➔&nbsp;`nonPaymentToken`         | `string`     | The result of our own card tokenization. Activated in POS for the merchant or merchant group.                                                                                                                                                                                                     |
 | └─➔&nbsp;`externalNonPaymentToken` | `string`     | The result of an external tokenization. This value will vary depending on card types, acquirers, customers, etc. For Mass Transit merchants, transactions redeemed by Visa will be populated with PAR. For Mastercard and Amex, it will be our own token. |
 | └➔&nbsp;`cardType`                | `string`  | `Credit Card` or `Debit Card`. Indicates the type of card used for the authorization.                                                                                                                                                                                                                |
-| └➔&nbsp;`maskedPan`               | `string`  | The masked PAN number of the card.                                                                                                                                                                                                                                                                   |
+| └➔&nbsp;`maskedPan`               | `string`  | The masked PAN number of the card.                                                                                                                                                                                                                                                                  |
+| └➔&nbsp;`maskedDPan`               | `string`  | The masked digital PAN number
+of the card. This is present if the payment was done with a tokenized card.                                                                                                                                                                                                                                                                   |
```

Comment:
Is "tokenized" card a known term for the integrators?

Reply:
Good question! I'll talk to a TOM or two to find out. If it is not a household term, I will add it to the terminology page and link it :)

Reply:
After a talk with POS, it seems to be a standardized term and the service is delivered by Visa and MasterCard. I think we can leave it be for now.